cmd/clusterctl/client/cluster/client.go

```diff
@@ -291,6 +264,20 @@ func newConnectBackoff() wait.Backoff {
 	}
 }
 
+// newShortConnectBackoff creates a new API Machinery backoff parameter set suitable for use when clusterctl connect to a cluster.
+// Preferred over newConnectBackoff() only when used to perform quick checks to check if a cluster is reachable.
+func newShortConnectBackoff() wait.Backoff {
+	// Return a exponential backoff configuration which returns durations for a total time of ~5s.
```

Comment:
nit, not sure if the total time is correct if the example is correct (feel free to ignore this comment)

Reply:
The example and the time are both correct. The example shows that the check is performed at the 0th second, 0.6th second, 0.25th second and so on. So the total time for this backoff will be ~5s.